src/shared_modules/content_manager/tests/component/action_test.cpp

```diff
@@ -107,6 +109,7 @@ TEST_F(ActionTest, TestInstantiationAndStartActionSchedulerForRawDataWithDeleteD
     // This file shouldn't exist because deleteDownloadedContent is enabled
     EXPECT_FALSE(std::filesystem::exists(downloadPath));
 
+    const auto contentPath {Utils::rightTrim(outputFolder + "/" + CONTENTS_FOLDER + "/3-" + fileName, ".xz")};
```

Comment:
Nitpick:
I find this a little clearer:
```suggestion
    const auto contentPath {outputFolder + "/" + CONTENTS_FOLDER + "/3-" + Utils::rightTrim(fileName, ".xz")};
```
If applied it should also be changed on the other occurrences.

Reply:
Done!